quesma/frontend_connectors/elasticsearch_ingest.go

```diff
@@ -0,0 +1,132 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+package frontend_connectors
+
+import (
+	"context"
+	"fmt"
+	"github.com/ucarion/urlpath"
+	"net/http"
+	quesma_api "quesma_v2/core"
+)
+
+type ElasticsearchIngestFrontendConnector struct {
```

Comment:
I would suggest to embed `frontend_connectors.BasicHTTPFrontendConnector` here and implement only `ServeHttp` or only part of it delegating common stuff to base implementation

```suggestion
type ElasticsearchIngestFrontendConnector struct {
  frontend_connectors.BasicHTTPFrontendConnector
```